src/main/java/com/lesson49/hibernateConfig/HibernateUtil.java

```diff
@@ -0,0 +1,46 @@
+package com.lesson49.hibernateConfig;
+
+import com.lesson49.entity.User;
+import org.hibernate.SessionFactory;
+import org.hibernate.boot.registry.StandardServiceRegistryBuilder;
+import org.hibernate.cfg.Configuration;
+import org.hibernate.cfg.Environment;
+import org.hibernate.service.ServiceRegistry;
+import org.springframework.context.annotation.*;
+import org.springframework.stereotype.Component;
+
+import java.util.Properties;
+
+
+@Component
+@ComponentScan(basePackages = "com.lesson49")
+public class HibernateUtil {
+    @Bean
+    public static SessionFactory getSessionFactory() {
+        try {
+            Configuration configuration = new Configuration();
+
+            Properties settings = new Properties();
+            settings.put(Environment.DRIVER, "org.postgresql.Driver");
```

Comment:
If you use java config, let's move these proeprties to application.properties file